runtime/src/iree/hal/cts/semaphore_submission_test.h

```diff
@@ -556,6 +427,8 @@ TEST_P(semaphore_submission_test, TwoBatchesWaitingOn1FormerBatchAmongst2) {
   //             ↓                        ↓
   //        semaphore21              semaphore22
 
+  printf("TwoBatchesWaitingOn1FormerBatchAmongst2 started.");
+
```

Comment:
Did you want to keep this debug print in?

Reply:
Thank you, I missed that. It is removed.